.github/renovate.json

```
@@ -1,42 +1,26 @@
 {
   "extends": [
-    "github>rancher/renovate-config#modigest"
+    "github>rancher/renovate-config#main"
```

All changes you rely on are now released:
```suggestion
    "github>rancher/renovate-config#release"
```